rules/technical-debt/rule.md

```diff
@@ -20,17 +22,19 @@ guid: de86d886-3341-43d0-a487-5e8b3cee3938
 ---
 ### What is Technical Debt?
 
-Technical Debt is when you take a shortcut to get a feature in to get some feedback.
+Technical Debt is when you defer work that needs doing in your code. And, just like when you defer a payment and acrue financial debt, technical debt _must_ be repaid, and it accumulates interest (in the form of reduced velocity) while it remains unpaid.
 
 `youtube: https://www.youtube.com/embed/ASVD4YIOgpU`
 
 <!--endintro-->
 
+Technical debt can occur for all kinds of reasons. The most common is when you take a shortcut or implement a hack to get a feature out quickly. Sometimes this is because, as a team (including the Product Owner), you've made a conscious decision o take this shortcut because, for example, you need a cut-down version of the feature urgently, or in other cases because of an open bug in a library you depend on.
```

Comment:
```suggestion
Technical debt can occur for all kinds of reasons. The most common is when you take a shortcut or implement a hack to get a feature out quickly. Sometimes this is because, as a team (including the Product Owner), you've made a conscious decision to take this shortcut because, for example, you need a cut-down version of the feature urgently, or in other cases because of an open bug in a library you depend on.
```